pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -396,6 +396,10 @@ def _postsolve(self):
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.optimal
+            if results.problem.sense == ProblemSense.minimize:
+                results.problem.lower_bound = results.solver.primal_bound
+            else:
+                results.problem.upper_bound = results.solver.primal_bound
```

Comment:
@whart222 I think the Jenkins failures are related to these lines. There are failures in the pickle tests and in the writer tests, all complaining that `primal_bound` is an unknown attribute on the `SolverInformation` class. As to why this only comes up on Jenkins and not GH actions, maybe @mrmundt or @jsiirola has a theory.

Reply:
Tests are all passing now.